pyproject.toml

```diff
@@ -19,7 +19,7 @@ dependencies = [
 ]
 
 classifiers=[
-    "License :: BSD License",
+    "License :: OSI Approved :: BSD License",
```

Comment:
Why did you add "OSI Approved"?

Reply:
testpypi rejected it without the OSI. I searched online and found that so i added it and it worked

Reply:
Ah ok! Thanks for checking!